app/commands.go

```diff
@@ -72,14 +73,15 @@ type unactivated struct {
 type activated struct {
 	cliBase
 
-	Status    statusCmd            `cmd:"" help:"Show status of Hermit environment." group:"env"`
-	Install   installCmd           `cmd:"" help:"Install packages." group:"env"`
-	Uninstall uninstallCmd         `cmd:"" help:"Uninstall packages." group:"env"`
-	Upgrade   upgradeCmd           `cmd:"" help:"Upgrade packages" group:"env"`
-	List      listCmd              `cmd:"" help:"List local packages." group:"env"`
-	Exec      execCmd              `cmd:"" help:"Directly execute a binary in a package." group:"env"`
-	Env       envCmd               `cmd:"" help:"Manage environment variables." group:"env"`
-	Validate  activatedValidateCmd `cmd:"" help:"Hermit validation." group:"global"`
+	Status      statusCmd            `cmd:"" help:"Show status of Hermit environment." group:"env"`
+	Install     installCmd           `cmd:"" help:"Install packages." group:"env"`
+	Uninstall   uninstallCmd         `cmd:"" help:"Uninstall packages." group:"env"`
+	Upgrade     upgradeCmd           `cmd:"" help:"Upgrade packages" group:"env"`
+	List        listCmd              `cmd:"" help:"List local packages." group:"env"`
+	Exec        execCmd              `cmd:"" help:"Directly execute a binary in a package." group:"env"`
+	Env         envCmd               `cmd:"" help:"Manage environment variables." group:"env"`
+	Validate    activatedValidateCmd `cmd:"" help:"Hermit validation." group:"global"`
+	LockDigests lockDigestsCmd       `cmd:"" help:"Lock the Digests for installed packages using sha256sums." group:"global"`
```

Comment:
```suggestion
	LockDigests lockDigestsCmd       `cmd:"" help:"Lock the manifest for installed packages." group:"global"`
```

Reply:
I think we are really locking in the digests and not the manifest. also again sha256sums is an existing attribute so that information is helpful.

Reply:
I'm not particularly fussed about the verbage; more that they should they be the same.